llama-index-integrations/vector_stores/llama-index-vector-stores-milvus/llama_index/vector_stores/milvus/utils.py

```diff
@@ -0,0 +1,40 @@
+from abc import ABC, abstractmethod
+from typing import List, Dict
+from FlagEmbedding import BGEM3FlagModel
+
+
+class BaseSparseEmbeddingFunction(ABC):
+    @abstractmethod
+    def encode_queries(self, queries: List[str]) -> List[Dict[int, float]]:
+        pass
+
+    @abstractmethod
+    def encode_documents(self, documents: List[str]) -> List[Dict[int, float]]:
+        pass
+
+
+class BGEM3SparseEmbeddingFunction(BaseSparseEmbeddingFunction):
```

Comment:
Its a little weird to call a class a function, but that might just be mean 😅

Reply:
We use the term `EmbeddingFunction` in pymilvus embedding utility, so it is consistent in context of Milvus.